e2e/test/iothub/service/DirectMethodE2ETests.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.Azure.Devices.E2ETests.Helpers;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.Azure.Devices.Client;
+
+namespace Microsoft.Azure.Devices.E2ETests.iothub.service
+{
+    [TestClass]
+    [TestCategory("E2E")]
+    [TestCategory("IoTHub")]
+    public class DirectMethodE2ETests : E2EMsTestBase
+    {
+        private readonly string _devicePrefix = $"{nameof(DirectMethodE2ETests)}_";
+        private readonly string _modulePrefix = $"{nameof(DirectMethodE2ETests)}_Module_";
+
+
+        [TestMethod]
+        [Timeout(TestTimeoutMilliseconds)]
+        public async Task DirectMethodsClient_InvokeAsync_MethodDoesNotExistAtFirst()
+        {
+            // arrange
+            using IotHubServiceClient serviceClient = TestDevice.ServiceClient;
+            var cts = new CancellationToken();
+
+            var methodInvocation = new DirectMethodServiceRequest("SetTelemetryInterval")
+            {
+                Payload = "10",
+                ResponseTimeout = TimeSpan.FromSeconds(30),
+            };
```

Comment:
In this case, the payload is not important.
```suggestion
            var methodInvocation = new DirectMethodServiceRequest("SomeMethod");
```